packages/lisk-transactions/src/index.ts

```diff
@@ -12,12 +12,12 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
+import { transfer } from './0_transfer';
 import { registerMultisignature } from './4_register_multisignature_account';
 import * as constants from './constants';
 import { createSignatureObject } from './create_signature_object';
 import {
```

Comment:
I just realized should also include `OutTransferClass` here too

Reply:
OutTransferClass is now being exported.